logistics_project/apps/malawi/warehouse/report_views/reporting_rate.py

```diff
@@ -18,6 +19,9 @@
 class View(warehouse_view.DistrictOnlyView):
 
     def custom_context(self, request):
+        if request.is_facility:
```

Comment:
i don't know how widespread this view class is used but if is_facility doesn't get set for each view then we should change this to `getattr(request, 'is_facility', False)`